packages/test-cases/cases/wrappers/wasm-rs/asyncify/src/lib.rs

```diff
@@ -1,118 +1,113 @@
 pub mod wrap;
-pub use wrap::*;
 use polywrap_wasm_rs::JSON;
 use wrap::imported::storage_module;
 use wrap::module;
+use wrap::module::{IModule, Module};
+pub use wrap::*;
 
-pub fn get_data(args: module::ArgsGetData) -> u32 {
-    match StorageModule::get_data(
-        &storage_module::ArgsGetData {},
-    ) {
-        Ok(v) => v.try_into().unwrap(),
-        Err(e) => panic!("{}", e),
+impl IModule for Module {
```

Comment:
Are we supporting both ways now? With IModule and without?
If so, we should have tests for the latter too.
Also, we should have the result return type for the latter as well

Reply:
Nope without IModule, it won't work. you need to implement all the methods from the IModule trait in Module.

Reply:
the older functions way is deprecated